xAPI.md

```diff
@@ -304,17 +304,24 @@ be unintuitive and/or lengthy to dissect into a list of requirements.
 
 ## 3.0 Guidenelines for Implementing Technical Specificationss Like xAPI
 
-## 4.0 Binding to JavaScript Object Notation (JSON)
+## 4.0 Serialization and JavaScript Object Notation (JSON)
 
-### 4.1 Serialization
+Serialization is the process of translating data objects and structures into a format for storage or transmission, such that the original data object can be recreated from the resulting serialization. In some cases it might be possible to serialize a piece of data in more than one way, for example a boolean property with a value of true might be represented as ```true``` or ```1``` depending on the serialization used. 
 
-### 4.2 Population of JSON Properties
+xAPI follows the rules of JSON for serializations (so boolean values are represented as ```true``` or ```false```). It might also be possible to represent the objects defined in this specification using other serializations, such as XML. This is out of scope of this dpecification and use of anything other than JSON to represent the objects defined in this specification is not conformant with this specification. 
```

Comment:
delete "with this specification" - not a big deal, but as I'm reviewing anyway...